src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -568,14 +579,9 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
             },
 
             text: {
-                condition: el => domUtils.isScriptElement(el) || domUtils.isStyleElement(el),
-
-                get: el => {
-                    if (domUtils.isScriptElement(el))
-                        return removeProcessingHeader(el.text);
+                condition: el => domUtils.isElementNode(el),
```

Comment:
Why you removed condition for scripts and styles here? Regular elements don't have this property and, therefore, we shouldn't process them. Though, it wasn't initially correct, since `<style>` element don't have this property as well.